library/include/ck/library/reference_tensor_operation/cpu/reference_gemm.hpp

```diff
@@ -64,8 +65,8 @@ struct ReferenceGemm : public device::BaseOperator
 
                 for(int k = 0; k < K; ++k)
                 {
-                    ADataType v_a;
-                    BDataType v_b;
+                    ComputeType v_a;
```

Comment:
How about using AccDataType instead of creating ComputeType?

Reply:
ComputeType is to match the design of mixed-precision kernels.